### DISCUSSION TITLE
New system font is blurry [Odin]

### DISCUSSION THREAD

**Discussion by stsdc** [elementary/default-settings#213] [General]
Testing right now Early Access build and just can't accommodate to the new system fonts. They are smaller and not so crisp as previous one in comparison to icons and terminal font which are really sharp. Eyes getting tired. Testing this on 23" 1920x1080. 
On the screenshot it looks better, though. :man_shrugging: 

![image](https://user-images.githubusercontent.com/6031763/105894790-8f996180-600c-11eb-8a04-22a2777d0f6d.png)
_Odin_
![obraz](https://user-images.githubusercontent.com/6031763/105921484-29770380-6039-11eb-910b-43f34492cc66.png)
_Hera_

It gives me 2012 Ubuntu font vibes :D IDK if there are any antyaliasing settings etc. But Hera don't have such a problem with a fonts.

**Comment by carniz** [comment]
I also see that on my Lenovo YogaSlim 7, with the 20210124 build of elementaryos-6.0-daily:

![Odin](https://user-images.githubusercontent.com/1648530/106034779-9d65ea00-60d3-11eb-9744-6999e197fb41.png)

For comparison, here's how the same plug looks in 5.1 Hera:

![Hera](https://user-images.githubusercontent.com/1648530/106034873-bbcbe580-60d3-11eb-98e2-b67fea1ef799.png)

Both tested in Live / "Try without install" mode.

(For some reason I also get English mixed with a language using Cyrillic characters (Russian?), but that's another topic)

**Comment by stsdc** [comment]
Can also confirm this issue on 1600x900 laptop display.
